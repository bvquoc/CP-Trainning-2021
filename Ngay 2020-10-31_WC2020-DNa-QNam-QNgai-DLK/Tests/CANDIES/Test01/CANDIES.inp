100
87630 1
23915 2
31187 3
55028 4
6248 5
34929 6
38943 7
76931 8
82273 9
11030 10
7012 11
13171 12
25990 13
96849 14
44562 15
52577 16
15071 17
62113 18
24717 19
97517 20
84214 21
70070 22
47922 23
87684 24
81975 25
63807 26
24055 27
21642 28
41078 29
84719 30
32500 31
59706 32
53314 33
16404 34
75502 35
84777 36
17097 37
84687 38
7451 39
3575 40
49644 41
40812 42
63270 43
12379 44
30673 45
87473 46
13901 47
74838 48
23525 49
53388 50
11425 51
23135 52
50717 53
65535 54
73867 55
81691 56
77946 57
66511 58
78563 59
57552 60
55401 61
84668 62
84631 63
31536 64
21273 65
11953 66
33769 67
24007 68
84035 69
69142 70
82947 71
34166 72
74669 73
50069 74
59329 75
91485 76
116 77
64885 78
28267 79
2023 80
10281 81
97876 25
32928 34
99250 23
42693 24290
46613 9
87238 33
82232 22
6889 43
73329 59566
68678 28
90711 38
92504 46
61641 7
88144 31
62435 30
47596 22764
89052 46
9525 13
70310 36