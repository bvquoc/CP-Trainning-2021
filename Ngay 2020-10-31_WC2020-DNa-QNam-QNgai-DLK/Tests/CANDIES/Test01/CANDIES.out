568897
137864
184289
338052
23960
152537
111262
159190
57510
10
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
21010
0
0
0
0
0
0
0
212460
0
0
0
0
0
0
